### PR TITLE
Changing 4.15 test to OVN

### PR DESCRIPTION
**Pull request by skordas**
https://issues.redhat.com/browse/PERFSCALE-2872

### CODE REVIEW

**Comment by skordas** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.15-nightly-x86-control-plane-private-24nodes pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.15-nightly-x86-control-plane-private-3nodes

**Comment by skordas** [comment]
@paigerube14 ptal

File: ci-operator/jobs/redhat-openshift-ecosystem/preflight/redhat-openshift-ecosystem-preflight-ocp-4.15-periodics.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 periodics:
 - agent: kubernetes
-  cluster: build02
+  cluster: build03
```
Comment (paigerube14) [inline ci-operator/jobs/redhat-openshift-ecosystem/preflight/redhat-openshift-ecosystem-preflight-ocp-4.15-periodics.yaml:3]: lets fetch the upstream and rebase with master, shouldnt be touching this folder/file

Comment (skordas) [reply ci-operator/jobs/redhat-openshift-ecosystem/preflight/redhat-openshift-ecosystem-preflight-ocp-4.15-periodics.yaml:3]: that was not changed by me - `make jobs` did that. I can redo this once again.

**Pull request merged**